.github/workflows/static_analysis.yml

```diff
@@ -1,24 +1,28 @@
 name: Static Analysis
-on: [push, pull_request]
+
+on:
+  push:
+  pull_request:
+    types: [opened]
 
 jobs:
   clang-tidy:
     name: Clang Tidy
     runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' && github.event.action == 'opened' && !github.event.pull_request.merged || github.event_name == 'push' }}
     container: ubuntu:rolling
     env:
       DEBIAN_FRONTEND: "noninteractive"
-      CONAN_USER_HOME: "${{ github.workspace }}/conan/"
-      CONAN_USER_HOME_SHORT: "${{ github.workspace }}/conan/s"
-      inexor_conan_path: "$HOME/.local/bin"
+      INEXOR_VULKAN_VERSION: "1.3.236.0"
```

Comment:
```suggestion
      INEXOR_VULKAN_VERSION: "1.3.215.0"
```

Reply:
I think now I should have fixed it in every place, I searched through everything with VS code. Done.

Reply:
@IceflowRE I have to revert these changes because this version of the Vulkan SDK does not exist. The Vulkan SDK is not updated with every Vulkan update apparently.

Reply:
We can use `1.3.216.0` though